src/components/Display.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ value }) => (<div>{value}</div>);
+
+Display.propTypes = {
+  value: PropTypes.string.isRequired,
+};
+
+export default Display;
```

Comment:
Also enable the default value (default props) to be '0'
--> https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png